backend/resources/center/main.css

```diff
@@ -0,0 +1,69 @@
+body {
+	background-color: transparent;
+	padding: 0;
+}
+
+.guide-list {
+	list-style: none;
+	padding: 0;
+}
+
+.guide-item {
+	width: 100%;
+	display: flex;
+	border: 1px solid transparent;
+	flex-direction: column;
+	padding-top: 8px;
+    padding-bottom: 8px;
+	color: var(--vscode-foreground);
+}
+.guide-item:hover {
+	background: var(--vscode-list-hoverBackground);
+}
+.guide-item.selected {
+	background: var(--vscode-list-inactiveSelectionBackground);
+	color: var(--vscode-foreground);
+}
+.guide-item.selected.focused {
+	background: var(--vscode-list-activeSelectionBackground);
+	color: var(--vscode-list-activeSelectionForeground);
+	border: 1px solid var(--vscode-focusBorder);
+}
+.guide-item.disabled {
+	background: var(--vscode-list-inactiveSelectionBackground);
+	color: var(--vscode-disabledForeground);
+}
+.guide-header {
+	width: 100%;
+	display: flex;
+	padding-bottom: 4px;
+}
+
+.guide-title {
+	display: block;
+	flex: 1;
+	width: 100%;
+	border: none;
+	padding: 0 0.6em;
+	font-weight: 700;
+	white-space: nowrap;
+	text-overflow: ellipsis;
+	overflow: hidden;
+	font-size: 13px;
+	line-height: 1.4em;
+}
+
+.guide-detail {
+	padding-right: 11px;
+	padding-left: 0.6em;
+	overflow: hidden;
+    text-overflow: ellipsis;
+	display: -webkit-box;
+	word-break: normal;
+	-webkit-box-orient: vertical;
+    -webkit-line-clamp: 2;
```

Comment:
Perhaps a minor formatting point: the indention of this line and 60th line is different from the other lines? (white space v.s. tab)

Reply:
Thanks, I ignored the format. I will use white space uniformly.